correios/client.py

```diff
@@ -298,7 +298,7 @@ def validate(self, document):
         schema = xml_utils.XMLSchema(xsd_document)
         return schema.assert_(document)
 
-    def get_xml(self, document) -> bytes:
+    def get_xml(self, document: object) -> object:
```

Comment:
I believe `object` doesn't work with python type annotations, I guess what you are trying to achieve here is this:

```python

from typing import Any

def get_xml(self, document: Any) -> bytes:
```